### ISSUE TITLE
Only get bids from Reviewers/Area Chairs that are in the Reviewers/Area Chairs group

### ISSUE THREAD

**Issue by carlosmondra** [openreview/openreview-expertise#44]
Currently we get bids from users that make bids. However, sometimes these users get removed from the Reviewers/Area Chairs group for various reasons. We should not get their bids in that case.

**Comment by melisabok** [comment]
Yes, this is a regular problem that we have in all the places we use the bids, for example: paper matching system. I'm not sure if the API should take care of this.

**Comment by carlosmondra** [comment]
> I'm not sure if the API should take care of this.

Yes, maybe it's a good idea to at least provide a parameter to only get the bids of the users that are still members of the reviewers/area chairs group.
